suites/squid/nvmeof/tier-2_8-1-nvmeof_cross-feature.yaml

```diff
@@ -1,7 +1,6 @@
-# Scale test suite including all new features from 8.1
-# 1 GW group with 4 GWs and 128 subsystems with 20 namespaces each and 5 initiator nodes
+# Test suite including all new features from 8.1
+# 1 GW group with 4 GWs and 5 subsystems with 10 namespaces each and 5 initiator nodes
 # Test conf at rhos-d conf/squid/nvmeof/ceph_nvmeof_ns-masking-5_client.yaml
-# Bare Metal conf TBD
 tests:
   - test:
       abort-on-fail: true
```

Comment:
In this test case, we are deploying OSDs on all available devices, does that mean this scenario considers NVMe collocated with OSDs on all NVMe nodes?
Or are we taking care of that in config file by giving nodes without free disks to deploy OSDs for NVMe, so that they won't be collocated?